rskj-core/src/test/java/co/rsk/peg/PendingFederationTest.java

```diff
@@ -246,8 +243,8 @@ void buildFederation_incomplete() {
 
     @Test
     void getHash() {
-        try (MockedStatic<BridgeSerializationUtils> bridgeSerializationUtilsMocked = mockStatic(BridgeSerializationUtils.class)) {
-            bridgeSerializationUtilsMocked.when(() -> BridgeSerializationUtils.serializePendingFederationOnlyBtcKeys(pendingFederation)).thenReturn(new byte[]{(byte) 0xaa});
+        try (MockedStatic<PendingFederation> pendingFederationMocked = mockStatic(PendingFederation.class)) {
+            pendingFederationMocked.when(() -> pendingFederation.serializeOnlyBtcKeys()).thenReturn(new byte[]{(byte) 0xaa});
```

Comment:
We could simplify this test without this mock, just create a PendingFederation object with certain keys and check against the expected result